### ISSUE TITLE
Header looks ugly on IE10

### ISSUE THREAD

**Issue by flavaflo** [axa-ch/style-guide#400]
maybe due to not fully supported flexbox things. also the button myAxa Login has a padding issue.

<img width="1426" alt="axa-styleguide-header-ie10" src="https://cloud.githubusercontent.com/assets/1135375/11237893/a225b18e-8de2-11e5-95b7-16ccb621c8be.png">

**Comment by davidknezic** [comment]
Yes, the problem is that everything is built with flexbox (more or less). I'm going to put all the header styles into the grid anyway (so it aligns nicely) and will try to make it nice in IE9+.

**Comment by sventschui** [comment]
For IE10 autoprefixer should make it work (adding -ms prefixed 2012 flexbox syntax).

Dor IE9 we'd need a polyfill

**Comment by davidknezic** [comment]
Have fun searching for an IE9 Flexbox polyfill. I'd rather recommend using floats and not making things visually perfect.

**Comment by sventschui** [comment]
Well then go for the "Browser not supported" as myAxa does

**Comment by AndyOGo** [comment]
@sventschui @davidknezic @flavaflo 
Can this be closed?
